### PR TITLE
Update who page

### PR DESCRIPTION
**Pull request by jfoucry**
Updating who's page.

### CODE REVIEW

File: content/page/who.en.md [hunk 1]
```diff
@@ -83,7 +85,14 @@ Exodus Privacy is a French (1901 law) non-profit organization. This organization
             <a href="https://github.com/Schoumi"><i class="fab fa-github fa-2x"></i></a>
             <p class="small">In charge of εxodus Android app development.</p>
         </div>
-    </div>
+    <!--</div>
```
Comment (codeurimpulsif) [inline content/page/who.en.md:88]: Can we remove this comments if they're not used?

File: content/page/who.fr.md [hunk 2]
```diff
@@ -83,7 +86,14 @@ Exodus Privacy est une association française à but non-lucratif, régie par la
             <a href="https://github.com/Schoumi"><i class="fab fa-github fa-2x"></i></a>
             <p class="small">En charge de l’application Android d’εxodus.</p>
         </div>
-    </div>
+    <!--</div>
```
Comment (codeurimpulsif) [inline content/page/who.fr.md:89]: Same here.

File: content/page/who.en.md [hunk 3]
```diff
@@ -36,24 +37,23 @@ Exodus Privacy is a French (1901 law) non-profit organization. This organization
...
         </div>
     </div>
     <div class="col-md-3 mt-3 text-center">
         <i class="far fa-3x fa-user mt-2 ml-auto mr-auto text-primary"></i>
         <div class="card-body">
-            <h6 class="card-title"><code>Gu1nn3zz</code></h6>
-            <a href="https://github.com/Gu1nness"><i class="fab fa-github fa-2x"></i></a>
-            <p class="small">In charge of εxodus development and translation.</p>
+            <h6 class="card-title"><code>Martoni</code></h6>
+            <a href="https://marmot.fr/@martioni"><i class="fab fa-2x fa-mastodon"></i></a>
```
Comment (martoni35) [inline content/page/who.en.md:47]: Hello,
Il y a une coquille sur le https://marmot.fr/@martioni qui est https://marmot.fr/@martoni

**Review by codeurimpulsif — CHANGES_REQUESTED** [review]
commented div break the page, please add them again, I can't update it myself because this PR is not on the exodus website repository.

**Comment by codeurimpulsif** [comment]
Moved to #129

**Pull request closed without merging**